### DISCUSSION TITLE
Automapper Many to many mapping

### DISCUSSION THREAD

**Discussion by wahidrezgui** [jasontaylordev/CleanArchitecture#554] [General]
Hy, 
Anyone has an idea how to add many to many mapping
I Added two Classes in the domain layer
 public class A
{
   public int IdA
    public ICollection<AB> ABs { get; set; }
}

public class B
{
 public int IdB
    public ICollection<AB> ABs { get; set; }
}

public class AB
{
    public int IdAB
    public int IdA
    public int IdB
    public A A { get; set; }
    public B B { get; set; }
}

and Now in the application layer  I want to Create ADtoVm Class
public class ADtoVm : IMapFrom<A>
{
    public ADtoVm ()
    {
        Bs= new List<B>();
    }

    public int IdA  { get; set; }
    

    public ICollection<B> Bs{ get; set; }

Now in the mapping Method I'm stacked
public void Mapping(Profile profile)
    {
profile.CreateMap<A, ADtoVm >()
               .ForMember(......., opt => opt.Ignore())
              .ForMember(dto => dto.Bs, opt => opt.MapFrom(x => x.........

I can't  figure out how to get the relation Many to many

as a result i want 
A {
    ..... properties
    Bs:[{  B1,B2}]
    
}
as a nested classes

**Comment by NessfertIndia** [comment]
Have you managed to do this? Or any workarounds available to map multiple classes to same dto
